omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/FulfillerDetailsResource2_2.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import io.swagger.models.Model;
+import io.swagger.models.ModelImpl;
+import io.swagger.models.properties.StringProperty;
+import org.openmrs.Order;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.docs.swagger.core.property.EnumProperty;
+import org.openmrs.module.webservices.rest.web.RequestContext;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.annotation.SubResource;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.api.PageableResult;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingSubResource;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+
+/**
+ * {@link Resource} for Order, supporting standard CRUD operations on fulfiller_comment and
+ * fulfiller_status
+ */
+@SubResource(parent = OrderResource2_2.class, path = "fulfillerdetails", supportedClass = FulfillerDetails.class, supportedOpenmrsVersions = {
+        "2.2.*" })
+public class FulfillerDetailsResource2_2 extends DelegatingSubResource<FulfillerDetails, Order, OrderResource2_2> {
+	
+	@Override
+	public FulfillerDetails newDelegate() {
+		return new FulfillerDetails();
+	}
+	
+	@Override
+	public FulfillerDetails save(FulfillerDetails delegate) {
+		Context.getOrderService().updateOrderFulfillerStatus(delegate.getOrder(), delegate.getFulfillerStatus(),
+		    delegate.getFulfillerComment());
+		return delegate;
+	}
+	
+	@Override
+	public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
+		DelegatingResourceDescription delegatingResourceDescription = new DelegatingResourceDescription();
+		delegatingResourceDescription.addProperty("fulfillerStatus");
+		delegatingResourceDescription.addProperty("fulfillerComment");
+		delegatingResourceDescription.addProperty("uuid");
```

Comment:
You don't need these properties because they actually don't exist nor apply

Reply:
Then I am not able to create the testcases for  FulfillerDetailsRessource2_2 because they would assume these variables. If you yet think that they are not needed I will remove them

Reply:
What happens when you exclude them?

Reply:
Then the tests in FulfillerDetailsResource2_2Test are failing e.g. https://github.com/openmrs/openmrs-module-webservices.rest/blob/e5e4c7d67cb0026d7ea99514fbad3e602f8a1f68/omod-common/src/test/java/org/openmrs/module/webservices/rest/web/resource/impl/BaseDelegatingResourceTest.java#L123

Reply:
That's what I was guessing, the correct thing to do in such cases is to override the tests and make sure they test for the correct thing i.e. that those fields do not exist since they are not include in the resource rep description.